### ISSUE TITLE
Google search console how to add astro paper to it

### ISSUE THREAD

**Issue by fazi-gondal** [satnaing/astro-paper#308]
Sir please write a detail article on how to add astro paper to Google search console please

**Comment by No1ceTea** [comment]
What do you mean by add documentation for Google Search Console ? Do you mean SEO ?

**Comment by maheshrijal** [comment]
@fazi-gondal you can refer to the google guide here: https://support.google.com/webmasters/answer/34592

**Comment by fazi-gondal** [comment]
> What do you mean by add documentation for Google Search Console ? Do you mean SEO ?

Yes because I don't know how to add astro paper theme to Google search console and do seo please tell me about it in detail

**Comment by fazi-gondal** [comment]
> @fazi-gondal you can refer to the google guide here: https://support.google.com/webmasters/answer/34592

Bro I don't know where i add Google search console verification tag in astro paper theme please tell me about it in detail because I want to add my website to Google search console and tell me little bit SEO

**Comment by satnaing** [comment]
Hello @fazi-gondal 
Did you manage to do that?
As @maheshrijal said, you can follow along with [the Google guide](https://support.google.com/webmasters/answer/34592).

You can verify your site ownership by providing Google site verification value in `PUBLIC_GOOGLE_SITE_VERIFICATION`. It is also mentioned in the [README](https://github.com/satnaing/astro-paper?tab=readme-ov-file#google-site-verification-optional).

AstroPaper also uses this way to verify the site ownership.

Here are the steps to follow:

### Step (1) Click `Add property` in Google Search Console.

### Step (2) Choose `URL prefix`

<img width="767" alt="Screenshot 2024-07-23 at 11 28 46 PM" src="https://github.com/user-attachments/assets/9ce88655-bc37-4326-94d5-8e61db449517">

### Step (3) Choose `HTML tag` method

![image](https://github.com/user-attachments/assets/4b9f4c2f-abf7-4f95-a7a8-45e04e19ed28)

### Step (4) Copy the content value from `<meta name="google-site-verification" content="copy-this" />` and put that value inside the `PUBLIC_GOOGLE_SITE_VERIFICATION` env variable.

### Step (5) Verify the HTML tag inside the search console.

Hope this helps. Let me know if you are still having any problems.

**Comment by satnaing** [comment]
I'm going to convert this issue to a discussion in case someone might need this in the future.
